core/modules/filter/filter.pages.inc

```diff
@@ -565,3 +571,41 @@ function _filter_get_file_id($href) {
   }
   return $result;
 }
+
+
+/**
+ * Forms a valid URL if possible from an entered address.
+ * An adaptation of function link_cleanup_url() from link module.
+ *
+ * Trims whitespace and automatically adds an http:// to addresses without a
+ * protocol specified
+ *
+ * @param string $url
+ *   The URL entered by the user.
+ * @param string $protocol
+ *   The protocol to be prepended to the URL if one is not specified.
+ *
+ * @return string
+ *   The URL with the protocol prepended if needed.
+ */
+function _filter_cleanup_url($url, $protocol = 'http') {
+  $url = trim($url);
+  $type = link_validate_url($url);
+  if(!$type){
```

Comment:
Need space after `if` and after closing bracket.

Reply:
Thanks.